.changeset/pre.json

```diff
@@ -0,0 +1,15 @@
+{
+  "mode": "pre",
+  "tag": "next",
```

Comment:
I don't know how it works but my intuition tells me we should be using this next tag instead of the alpha one? At least that's what they even describe in the changesets documentation